### ISSUE TITLE
FR: tab rule for lists

### ISSUE THREAD

**Issue by Friedolito** [platers/obsidian-linter#1410]
Sometimes pasted text as list that use 2 spaces for indentation. 
I set my rule in the editor to 4 spaces, what I find better to read and work with.
Manually adding spaces to a long list is cumbersome and would be a great usecase for a linter

## Describe the Solution You'd Like

Add an option to set a rule for indentation with in lists. ( one indentation = 4 spaces)

If a list with different indentation is detected, change to the rule. 

``` markdown
current 
- item
  - item
  - item
    - item 
```

``` markdown
expected outcome
- item
    - item
    - item
        - item
```

Could be achieved using a linter in vs code but would be great to have it in obsmd, especially because linter is already in my workflow

Thank you for considering and creating linter.
